packages/backend/src/database/migrations/20230620204726_add_pie_to_chart_type.ts

```diff
@@ -0,0 +1,10 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex('chart_types').insert({ chart_type: 'pie' });
+}
+
+export async function down(knex: Knex): Promise<void> {
+    await knex('saved_queries_versions').where('chart_type', 'pie').delete();
+    await knex('chart_types').delete().where('chart_type', 'pie');
```

Comment:
won't this delete versions for you on cascade?

Reply:
no, table definition does not have constraints
<img width="332" alt="CleanShot 2023-06-22 at 13 29 19@2x" src="https://github.com/lightdash/lightdash/assets/962095/a7e9b640-f06b-4e97-b066-088943262772">

Reply:
Sorry I meant on `saved_queries_versions` but actually I can see it's `RESTRICT` ! Never mind

https://github.com/lightdash/lightdash/blob/main/packages/backend/src/database/migrations/20210728093219_saved_queries.ts#L134-L140

Reply:
you can resolve this